### ISSUE TITLE
Platform: Editor comment button is outside of the editor

### ISSUE THREAD

**Issue by Sebastian-ubs** [BiblioNexus-Foundation/scripture-editors#127]
With the comment button (that appears on double clicking a word) overflowing the editor by 17px, there is a need to provide for this space when embedding the editor anywhere.
While not using comments, it is unclear why this space is preserved - especially when this space has a (slightly) different background color.
![grafik](https://github.com/BiblioNexus-Foundation/scripture-editors/assets/141921979/55be61a7-a0dc-41ed-9596-2e85add0739d)

Unfortunately this space is as well not usable for / not used by the toolbar
![grafik](https://github.com/BiblioNexus-Foundation/scripture-editors/assets/141921979/d904b127-1a0d-443e-95e4-215bc2050afa)

I understand the inspiration by google docs, but there is always a lot of space around it and it is not ment to be put into a lot of small editor windows (like for Platform).
So: How to solve this? Should the button position be configurable? Or there always should be space (in the same color) provided? Or ... ?

**Comment by irahopkinson** [comment]
@Sebastian-ubs yeah the editor is still pre-release so it doesn't claim to have many of these things sorted. I've added the commenting functionality from the Lexical Playground almost verbatim. The functionality is there to spark this kind of discussion both for UX and how it interacts with Platform.Bible including designing the comment data model. I'm still waiting to hear from UX about all the kinds of things you have raised above and more.

**Comment by Sebastian-ubs** [comment]
Thanks @irahopkinson, was there any specific request to UX about reviewing certain things of the editor? I have not heared about it.

**Comment by Sebastian-ubs** [comment]
btw: I was able to work around it using
```css
.CommentPlugin_AddCommentBox {
  left: unset !important;
  right: 0 !important;
}
```
It seems strange that positioning something at the right border is done using the `left` property anyhow.

I am curious to hear what you think should be the next steps to discuss the comment data model and visual appearance.

**Comment by yonacwy** [comment]
Try react-responsive and flex options in css.


![Image](https://github.com/user-attachments/assets/bb7645a9-a8d5-43d9-8197-5a1f0a798a63)
![Image](https://github.com/user-attachments/assets/db3ce996-a081-465b-b490-799f0fb8407a)
